src/main/java/com/dev/cinema/controller/AuthenticationController.java

```diff
@@ -19,7 +20,7 @@ public AuthenticationController(AuthenticationService authenticationService) {
     }
 
     @PostMapping("/register")
-    public void registration(@RequestBody UserDtoRequest request) {
```

Comment:
let's remove this class as well if it is not removed yet
`UserDtoRequest`

Reply:
Removed